ui/.eslintrc

```diff
@@ -1,7 +1,10 @@
 {
     "extends": ["react-app","standard"],
+    "plugins": ["react-hooks"],
     "rules": {
         "semi": [2, "always"],
-        "prefer-const": [2]
+        "prefer-const": [2],
+        "react-hooks/rules-of-hooks": "error",
+        "react-hooks/exhaustive-deps": "warn"
```

Comment:
Would there be an additional benefit of setting this level to "error"?

There are warnings in the code already, so we may miss them now because they are done. With it set to "error" we can ensure we find and fix them.

Reply:
T

Reply:
These are the recommendations from react